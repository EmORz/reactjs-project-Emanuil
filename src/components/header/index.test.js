import renderer from 'react-test-renderer'
import Header from '.'
import React from 'react'

describe('Header componenents', ()=>{
    it('shoud render authenticate router', ()=>{

const tree = renderer.create(<Header/>).toJSON()
expect(tree).toMatchSnapshot()
    })
     it('shoud render unauthenticate router', ()=>{

const tree = renderer.create(<Header/>).toJSON()
expect(tree).toMatchSnapshot()
    })

})