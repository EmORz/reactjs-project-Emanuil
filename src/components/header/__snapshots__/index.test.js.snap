// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header componenents shoud render authenticate router 1`] = `
<header
  className="navigation"
>
  <ul>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/register"
      >
        Register
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/login"
      >
        Login
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/about"
      >
        About
      </a>
    </li>
  </ul>
</header>
`;

exports[`Header componenents shoud render unauthenticate router 1`] = `
<header
  className="navigation"
>
  <ul>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/register"
      >
        Register
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/login"
      >
        Login
      </a>
    </li>
    <li
      className="header-list-item"
    >
      <a
        className="header-link"
        href="/about"
      >
        About
      </a>
    </li>
  </ul>
</header>
`;
